### PR TITLE
Adds AOS to About ---> Devices page

### LINKED ISSUE #1126: Remove AOS on Devices page
The AOS on the Devices page should be removed in order to keep it consistent with https://github.com/fossasia/chat.susi.ai/issues/1124

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Akshat-Jain**
Fixes #1126 

Changes: Adds AOS effects to the different sections of the About ---> Devices page.

Demo Link: https://pr-1126-fossasia-susi-web-chat.surge.sh

### CODE REVIEW

File: src/components/Devices/Devices.react.js [hunk 1]
```diff
@@ -8,6 +8,9 @@ import PropTypes from 'prop-types';
 import StaticAppBar from '../StaticAppBar/StaticAppBar.react';
 import $ from 'jquery';
 import Footer from '../Footer/Footer.react';
+import "animate.css/animate.min.css";
+import ScrollAnimation from 'react-animate-on-scroll';
+<link rel="stylesheet" href="bower_components/aos/dist/aos.css" />
```
Comment (duttaditya18) [inline src/components/Devices/Devices.react.js]: Use `import` i guess.

File: src/components/Devices/Devices.react.js [hunk 2]
```diff
@@ -67,28 +81,35 @@ class Devices extends Component{
             </p>
             <div className="store_icons">
               <Link className="app_store"
-                target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store
+                target="_blank" to="https://github.com/fossasia/susi_iOS"> Download on the App Store"
```
Comment (duttaditya18) [inline src/components/Devices/Devices.react.js]: Random `"`?

**Comment by Akshat-Jain** [comment]
@isuruAb Fixed the travis errors. Please review :-)

File: src/components/Devices/Devices.react.js [hunk 3]
```diff
@@ -30,9 +33,12 @@ class Devices extends Component{
     return (
       <div>
 
+      <script src="bower_components/aos/dist/aos.js"></script>
```
Comment (anshumanv) [inline src/components/Devices/Devices.react.js]: This simply doesn't go here.

**Comment by Akshat-Jain** [comment]
@isuruAb Can you please review and merge ? :)

**Comment by Akshat-Jain** [comment]
@isuruAb Please merge :)

**Pull request merged**